Misc/NEWS.d/next/Library/2018-07-30-12-00-15.bpo-31658._bx7a_.rst

```diff
@@ -0,0 +1 @@
+Make xml.sax.parse accepting Path objects. Patch by Mickaël Schoentgen.
```

Comment:
Markup string can be used for linking. How about the below


:func:`xml.sax.parse` now supports :term:`path-like object`.
Patch by Mickaël Schoentgen.

Reply:
Could you double-check that this is true for NEWS entries?

Reply:
@merwok Yes, just verified the links with a local build. Screenshot 

![screen shot 2019-01-08 at 9 48 06 pm](https://user-images.githubusercontent.com/3972343/50843919-a7eba480-138f-11e9-8ec1-1a769e6e1cc0.png)

Reply:
Done :+1: